translations/ui-rs/en_US.json

```diff
@@ -10,6 +10,12 @@
   "settings.general.message": "These are your general app settings.",
   "settings.some-feature": "Some Feature",
   "settings.some-feature.message": "These are your settings for some app feature.",
+  "settings.supply-address-settings": "Addresses",
+  "settings.supply-address-settings-title": "Address Settings",
+  "settings.supply-address-settings-list": "Address List",
+  "settings.supply-address-settings.singular": "Address",
+  "settings.supply-address-settings.name": "Name",
+  "settings.supply-address-settings.value": "Value",
```

Comment:
The names of these translation tags seem redundant to me: wouldn't `settings.supply-address.WHATEVER` suffice?

Reply:
I agree, looking back at these the names seem like bizarre choices... I've fixed those now

Reply:
Apologies for the capitals, probably should have picked up on that by now. Fixed that now.